packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1363,7 +1370,7 @@ export default class TradeStore extends BaseStore {
 
         // Set stake to default one (from contracts_for) on symbol or contract type switch.
         // On contract type we also additionally reset take profit
-        if (this.default_stake && this.is_dtrader_v2_enabled) {
+        if (this.default_stake && (this.is_dtrader_v2_mobile || this.is_dtrader_v2_desktop)) {
```

Comment:
Same here. Many please, where you check `this.is_dtrader_v2_mobile || this.is_dtrader_v2_desktop`. Create a getter to check them both?